php.md

```diff
@@ -0,0 +1,11 @@
+# PHP on Graviton
```

Comment:
Let's add an intro and a couple of points about other distributions. Something like: "PHP 7.4 and later are tested to perform well on Graviton. It works out of the box on Ubuntu 22.04 and AL2023, but requires an extra step on AL2."

Reply:
Added